MonoGame.Framework/FrameworkDispatcher.cs

```diff
@@ -2,15 +2,57 @@
 // This file is subject to the terms and conditions defined in
 // file 'LICENSE.txt', which is part of this source code package.
 
+using System;
+using Microsoft.Xna.Framework.Audio;
+
 namespace Microsoft.Xna.Framework
 {
-	public static class FrameworkDispatcher
-	{
-		public static void Update()
-		{
-			// "Updates the status of various framework components
-			// (such as power state and media), and raises related events"
-		}
-	}
+    /// <summary>
+    /// Helper class for processing internal framework events.
+    /// </summary>
+    /// <remarks>
+    /// If you use <see cref="Game"/> class, <see cref="Update()"/> is called automatically.
+    /// Otherwise you must call it as part of your game loop.
+    /// </remarks>
+    public static class FrameworkDispatcher
+    {
+        private static bool _initialized = false;
+
+        /// <summary>
+        /// Processes framework events.
+        /// </summary>
+        public static void Update()
+        {
+            if (!_initialized)
+                Initialize();
+
+            DoUpdate();
+        }
+
+        private static void DoUpdate()
+        {
+            DynamicSoundEffectInstanceManager.UpdatePlayingInstances();
+            SoundEffectInstancePool.Update();
+        }
+
+        private static void Initialize()
+        {
+            // Initialize sound system
+#if DIRECTX
+            SoundEffect.InitializeSoundEffect();

```

Comment:
So the only issue that we need to fix here is this `#if` block.  If we commit as is then we break the console repos.  We need to add a platform method into the `SoundEffect` partial class like:  `SoundEffect.PlatformInitializeSystem()` which can then be separately implemented for XAudio and OpenAL platforms.

This way consoles can correctly overload this behavior for their respective audio systems.

If you don't have the time to do this @polsys let me know.